shared/rakefiles/xk.rake

```diff
@@ -65,6 +65,37 @@ task :xk, [:cmd, :skip_infra, :skip_secret_mgmt] => [@serviceaccount_key_file, @
   sh_filter "#{@exekube_cmd} #{args[:cmd]}" if args[:cmd]
 end
 
+task :refresh_infra, [:project_type] => [@gcp_creds_file] do | taskname, args|
+
+  next if args[:project_type] == "common"
+
+  # The common infrastructure might create the DNS resources, in order to avoid
+  # conflicts at the creation resources we need to import the already created
+  # DNS zone.
+
+  dns_zone = @env + "-" + args[:project_type] + "-" + ENV["TF_VAR_organization_domain"].tr(".", "-")
```

Comment:
Since you're doing this calculation in Ruby code as well as in several places in Terraform code, how about doing it once in `set_vars()` and referring to the result of that calculation where needed?